test/e2e/load.go

```diff
@@ -232,22 +285,30 @@ func computeRCCounts(total int) (int, int, int) {
 	return smallRCCount, mediumRCCount, bigRCCount
 }
 
-func generateRCConfigs(totalPods int, image string, command []string, c *client.Client, nss []*api.Namespace) []*framework.RCConfig {
+func generateRCConfigs(totalPods int, image string, command []string, nss []*api.Namespace) []*framework.RCConfig {
 	configs := make([]*framework.RCConfig, 0)
 
 	smallRCCount, mediumRCCount, bigRCCount := computeRCCounts(totalPods)
-	configs = append(configs, generateRCConfigsForGroup(c, nss, smallRCGroupName, smallRCSize, smallRCCount, image, command)...)
-	configs = append(configs, generateRCConfigsForGroup(c, nss, mediumRCGroupName, mediumRCSize, mediumRCCount, image, command)...)
-	configs = append(configs, generateRCConfigsForGroup(c, nss, bigRCGroupName, bigRCSize, bigRCCount, image, command)...)
+
+	// Create a number of clients to better simulate real usecase
+	// where not everyone is using exactly the same client.
+	clients, err := createClients((smallRCCount + 19) / 20)
```

Comment:
summing up small + medium + big doesn't make much sense, since we won't use more than first "small" anyway (those arrays are separate, so it will require changing other things, which would be unnecessary complication in my opinion).

Introduced constant.1